Automattic/woocommerce-payments
# Only output single currency breadcrumbs when rendering the settings page

Fixes #2282

#### Changes proposed in this Pull Request

The `save` method calls `get_settings`, causing the breadcrumbs to be rendered also when submitting the single currency settings form.

This PR adds a check to prevent rendering the breadcrumbs when not rendering the settings page, avoiding the warning and extra breadcrumbs at the top of the page.









#### Testing instructions





* Navigate to the settings page for a single currency
* Make sure the breadcrumbs show up before the fields
* Update the form and submit it
* Make sure no breadcrumbs or warnings show up at the top of the page
* Make sure the breadcrumbs are displayed before the setting fields

-------------------
